# Copyright 2008 Sun Microsystems, Inc.  All rights reserved.
# Use is subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
Change .so version to match existing Solaris libraries, since there's
no incompatible change since libFS.so.5

diff -urp -x '*~' -x '*.orig' src/Makefile.am src/Makefile.am
--- src/Makefile.am	2008-05-21 17:27:09.000000000 -0700
+++ src/Makefile.am	2008-05-21 19:03:29.651500000 -0700
@@ -35,7 +35,7 @@ libFS_la_LIBADD = $(FS_LIBS) $(XTRANS_LI
 
 INCLUDES = -I$(top_srcdir)/include/X11/fonts
 
-libFS_la_LDFLAGS = -version-number 6:0:0 -no-undefined
+libFS_la_LDFLAGS = -version-number 5 -no-undefined
 
 libFSincludedir = $(includedir)/X11/fonts
 libFSinclude_HEADERS = \
